metaflow/plugins/pypi/bootstrap.py

```diff
@@ -116,7 +121,9 @@
         # Install PyPI packages.
         cmds.extend(
             [
-                f"""./micromamba run --prefix {prefix} pip --disable-pip-version-check install --root-user-action=ignore --no-compile {pypi_pkgs_dir}/*.whl"""
+                f"""set -e;
+                export PATH=$PATH:$(pwd)/micromamba;
```

Comment:
ditto here.

Reply:
yes - these are executed within different shells right now - they can be moved into a single shell command as well as a quick follow up.